toptal/keycodes
# Update public repo with private repo code

[UTI-2483](https://toptal-core.atlassian.net/browse/UTI-2483)

### Description

This PR will add into `keycodes-app` all progress from the private repo without sensitive info. Please check for that.

### How to test

- `yarn && yarn build && yarn start`
- check that everything is running properly - including tests

### Pre-merge checklist

- [x] Verify that feature branch is up-to-date with `main` (if not - rebase it).
